package.json

```diff
@@ -0,0 +1,20 @@
+{
+  "name": "thrift-pool",
+  "version": "1.0.0",
+  "description": "A module that wraps thrift interfaces in connection pooling logic to make them more resilient.",
+  "main": "index.js",
```

Comment:
No the index.js file includes coffeescript and the rest of the coffee source - this is a well established pattern to avoid pre publish compilation steps. See https://github.com/Clever/quest/blob/master/index.js